mcstatus/scripts/mcstatus.py

```diff
@@ -57,14 +57,14 @@ def status():
     servers that are version 1.7 or higher.
     """
     response = server.status()
-    click.echo("version: v{} (protocol {})".format(response.version.name, response.version.protocol))
+    click.echo("version: v{} (protocol {})".format(response.version.name.encode("utf-8"), response.version.protocol))
```

Comment:
This change causes the version to become surrounded with quotes and prefixed with a `b`:
```
└─▪ mcstatus my.server status
version: vb'1.15.2' (protocol 578)
description: "{'text': 'A Fabric Minecraft Server powered by Docker'}"
players: 0/20 No players online
```
Can you explain, or should this part be changed back?